docs/DOCUMENTATION_GUIDE.md

```diff
@@ -0,0 +1,22 @@
+# Documentation Guidelines
+
+## Starting a New Project
+
+The dashboards-anywhere repository hosts multi-projects, this documentation guidelines provides documentation guide for the developers who starting a new project. 
```

Comment:
Minor grammar: 

`The dashboards-anywhere repository hosts multi-projects. The documentation guidelines are for the developers who start a new project. `

Reply:
I already merged the changes when saw this comment, will address it in the next PR.

Reply:
This [PR](https://github.com/opensearch-project/dashboards-anywhere/pull/20), it will address your comment